packages/renderer/src/components/molecules/MarketBuySellSwitch/MarketBuySellSwitch.tsx

```diff
@@ -0,0 +1,82 @@
+// =============================================================================
+//  Copyright 2022 Haveno
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// =============================================================================
+
+import type { TabsProps } from "@mantine/core";
+import { Tabs } from "@mantine/core";
+
+export function MarketBuySellSwitch(props: TabsProps) {
```

Comment:
If you looked to the component deeply it's not switch at all, switch something you can turn it on/off but that component is tabs something you can move its state/view which is completely different. In terms of putting that component in generic place don't see that component would be reusable again and that design is limited to the filter bar only.

Reply:
We need a generic component that lets us select between multiple states. Call it what you will, in UX parlance it still is a "switch". Ours would be a bit more generic as in it can be more than a boolean toggle by supporting more than two states if required. Bottomline: 
1. we want it to be generic
2. "MarketBuySellSwitch" isn't a good name
3. internals should be abstracted away